test/build.js

```diff
@@ -139,6 +139,22 @@ describe('grunt', function() {
         done();
       });
     });
+
+    // Ensure the build/html/config directory exists if it should.
+    it('config directory should exist if provided ', function(done) {
+      var src = fs.existsSync('src/config');
+      var html = fs.existsSync('build/html/config');
+      if (src && html) {
+        assert.ok(true, 'Both src/config and build/html/config exist');
+        done();
+      } else if (src && !html) {
+        assert.ok(false, 'Error: src/config exists but build/html/config does not');
+        done();
+      } else if (html && !src) {
+        assert.ok(false, 'Error: build/html/config exists but src/config does not');
+        done();
```

Comment:
I appreciate the thoroughness here. Do we have Drupal 7 test cases executing that won't have the config directory? If so, that seems like it deserves a separate test case.